tests/test_authsidecar.py

```diff
@@ -14,7 +19,9 @@ def test_authSidecar_alertmanager(self, kube_version):
         """Test Alertmanager Service with authSidecar."""
         docs = render_chart(
             kube_version=kube_version,
-            values={"global": {"authSidecar": {"enabled": True}}},
+            values={
+                "global": {"authSidecar": {"enabled": True, "tag": "placeholder-tag"}}
```

Comment:
This test has no assertions against this tag.

Reply:
removed the extraneous tag